src/Database/LSMTree/Internal/Index/Compact.hs

```diff
@@ -548,13 +540,6 @@ putPaddingTo64 written
   Deserialisation
 -------------------------------------------------------------------------------}
 
--- | The input bytestring must be 64 bit aligned and exactly contain the
--- serialised compact index, with no leading or trailing space.
--- It is directly used as the backing memory for the compact index.
---
--- Also note that the implementation reads values in little-endian byte order.
--- If the file has been serialised in big-endian order, the mismatch will be
--- detected by looking at the type–version indicator.
```

Comment:
I don't know how to better solve this, but it seems unfortunate that we lose documentation on these functions, even if the documentation is moved there. Maybe link to where the documentation is now found?

Reply:
I’ve moved this documentation to the class now and improved it along the way. The corresponding changes have been introduced by 559cc181bf7fd7114aba70d3c1604ab9000b8a52. Since 3141593ede80ab3ad5e649bdf0ece9286cd936ae has added a reference to the documentation in the class, the problem you’re mentioning here should be solved.